src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -389,26 +368,25 @@ export class CalcitePopover {
   render() {
     const { _referenceElement, open, disablePointer } = this;
     const displayed = _referenceElement && open;
+    const arrowNode = !disablePointer ? (
+      <div class={CSS.arrow} ref={arrowEl => (this.arrowEl = arrowEl)}></div>
+    ) : null;
 
     return (
       <Host
         role="dialog"
+        class={{
+          [hydratedInvisibleClass]: !displayed
```

Comment:
Could you double check this doesn't clobber user-defined classes?

Reply:
verified: 
![image](https://user-images.githubusercontent.com/1231455/74177063-d6b8f780-4bed-11ea-858c-79aed3c4dc31.png)

✅

Reply:
👍